labs/advanced-topics/consumer/build-image.sh

```diff
@@ -1,2 +1,7 @@
-#! /bin/bash
```

Comment:
Adding these set directives means the scripts will stop running after the first error happens, rather than hiding the error which is what happens without them. More info here: https://medium.com/expedia-group-tech/using-bash-for-devops-7046eed1aa63